testing/jormungandr-integration-tests/src/jormungandr/bft/mempool/v0.rs

```diff
@@ -0,0 +1,235 @@
+use crate::common::jormungandr::{ConfigurationBuilder, Starter};
+use crate::common::startup;
+use assert_fs::TempDir;
+use jormungandr_lib::interfaces::BlockDate;
+use jormungandr_lib::interfaces::InitialUTxO;
+use jormungandr_lib::interfaces::Mempool;
+use jormungandr_testing_utils::testing::node::time;
+use jormungandr_testing_utils::testing::{FragmentSenderSetup, FragmentVerifier};
+use std::time::Duration;
+
+#[test]
+pub fn test_mempool_pool_max_entries_limit() {
+    let temp_dir = TempDir::new().unwrap();
+
+    let receiver = startup::create_new_account_address();
+    let mut sender = startup::create_new_account_address();
+
+    let leader_config = ConfigurationBuilder::new()
+        .with_funds(vec![
+            InitialUTxO {
+                address: sender.address(),
+                value: 100.into(),
+            },
+            InitialUTxO {
+                address: receiver.address(),
+                value: 100.into(),
+            },
+        ])
+        .with_explorer()
+        .with_slot_duration(2)
+        .with_mempool(Mempool {
+            pool_max_entries: 1.into(),
+            log_max_entries: 100.into(),
+        })
+        .build(&temp_dir);
+
+    let jormungandr = Starter::new()
+        .config(leader_config.clone())
```

Comment:
If you add the `temp_dir` to the starter all configs and files will be saved in case of failure
```
Starter::new().temp_dir(temp_dir)...
```
This applies to all other tests as well